test/sql/function/timestamp/make_date.test

```diff
@@ -217,6 +217,12 @@ SELECT make_timestamp(0), make_timestamp(1684509234845000);
 ----
 1970-01-01 00:00:00	2023-05-19 15:13:54.845
 
+# From nanoseconds
+query II
+SELECT make_timestamp_ns(0), make_timestamp_ns(1684509234845000123);
+----
+1970-01-01 00:00:00	2023-05-19 15:13:54.845000123
+
```

Comment:
Should we add tests for `min` and `max` values?

Reply:
Do you mean `make_timestamp_ns(-9,223,372,036,854,775,808)` and `make_timestamp_ns(9,223,372,036,854,775,807)` ?

Reply:
I've added a test for the maximum values. 

For negative values, I've ran into some problems:
`make_timestamp_ns(-1)` triggers an assertion:
```
select make_timestamp_ns(-1);
INTERNAL Error:
Information loss on integer cast: value -952 outside of target range [-128, 127]
```
I couldn't immediately figure out why, or how to get a stack trace for the respective exception.

`select make_timestamp(-9223372036854775807)` correctly says `-infinity`.

`make_timestamp_ns(-9223372036854775806)` says:
```
select make_timestamp_ns(-9223372036854775806);
Conversion Error:
Date out of range in timestamp_ns conversion
```
This is similar to `make_timestamp_ns(-9223372036854775806)`. I don't know if the behavior is good; I couldn't find a lower limit for `make_timestamp()` documented anywhere.

Would it be acceptable to reject any negative argument to `make_timestamp_ns()` ? I'll kindly take advice for how to proceed / if you could look into the assertion error if rejecting negative input is not acceptable, I'd appreciate it.